app/build.gradle

```diff
@@ -51,6 +55,7 @@ dependencies {
     compile 'com.android.support:design:25.3.1'
     compile 'com.google.code.gson:gson:2.7'
     compile 'org.greenrobot:eventbus:3.0.0'
+    compile 'cn.pedant.sweetalert:library:1.3'
```

Comment:
Have you consider using the Android build-in [Dialogs](https://developer.android.com/guide/topics/ui/dialogs.html)/[PopupWindows](https://developer.android.com/reference/android/widget/PopupWindow.html). I'm in favor of using these over some custom library for consistency with other Android apps...